guides/user/kubernetes-v2/best-practices/index.md

```diff
@@ -43,45 +43,48 @@ To keep track of what's changing in your cluster, and under what circumstances
 it is changing, we highly recommend [configuring Spinnaker to emit events to an
 external audit
 log](https://blog.spinnaker.io/spinnaker-echo-google-cloud-functions-stackdriver-logging-spinnaker-audit-log-81139f084db9).
-Spinnaker will already generate events for any events running through its
+
+Spinnaker already generates events for any events running through its
 orchestration engine, all that's needed is an endpoint to send them to.  When
 set up, any pipeline that deploys or updates a manifest will have your fully
 hydrated manifest recorded.
 
 ## Version Your ConfigMaps and Secrets
 
-Anytime Spinnaker deploys a ConfigMap or Secret, it will append a version to
-its name. If that exact ConfigMap or Secret is already running in the cluster,
-it won't be redeployed. Downstream stages that reference that ConfigMap or
-Secret will deploy the version chosen by Spinnaker. __Unless your application
+Anytime Spinnaker deploys a ConfigMap or secret, it appends a version to
```

Comment:
Secret was capitalized because it is a Kubernetes resource kind in the same that ConfigMap is.